docs/Glossary/index.md

```diff
@@ -0,0 +1,259 @@
+
+### [Lidarr](../Lidarr/index.md)
+
+An application designed to help with the automation and management of your music library. One of a group of applications, including Prowlarr, Radarr, and Sonarr, collectively known as the "arrs".
+
+---
+
+## M
+
+---
+
+## N
+
+### Notice & Takedown
+
+!!!info
+
+    Also known as N&T
+
+The European equivalent of [DMCA](#dmca).
+
+[More info from Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}
+
+---
+
+## O
+
+---
+
+## P
+
+### PAR (PAR2) Files
+
+PAR and PAR2 files, or 'Parity Archive Volume' files, are primarily used in Usenet to help repair incomplete or corrupted files. However, their presence does not always guarantee successful reconstruction of the original data.
+
+### Peer
+
+A general term that collectively refers to both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Perma-Seed
+
+Seeding your torrents permanently ("forever"). Most often this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.
+
+### [Prowlarr](../Prowlarr/index.md)
+
+An application designed to help with the automation and management of your [trackers](#tracker) and [indexers](#indexer). One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".
+
+---
+
+## Q
+
+### qBit Manage
+
+[qBit Manage](https://github.com/StuffAnThings/qbit_manage){:target="\_blank" rel="noopener noreferrer"} (also known as `qbm` or `qbitmanage` / `qb_manage`) is a tool for managing your [qBittorrent](#qbittorrent) instance. It automates actions on torrents meeting specified criteria, such as time seeded and tracker.
+
+### qBittorrent
+
+!!!info
+
+    Also known as: qb, qbit, qbittorrent
+
+[qBittorrent](https://www.qbittorrent.org/){:target="\_blank" rel="noopener noreferrer"} is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.
+
+### Quality
+
+Refers to the specific quality of a release, describing both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+!!!info
+
+    Also known as: QP
+
+A defined group, and preferential ordering, of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
```

Comment:
inconsistencies for `TV shows` and `TV Shows`

Reply:
I think these are OK. One is referring to TV Shows in the plural, and the other is referring to a singular TV Show Library.